src/core/core.scale.js

```diff
@@ -670,6 +683,31 @@ var Scale = Element.extend({
 		return rawValue;
 	},
 
+	_convertTicksToLabels: function(ticks) {
+		var me = this;
+		var labels, i, ilen;
+
+		me.ticks = ticks.map(function(tick) {
+			return tick.value;
+		});
```

Comment:
I dislike this, but I guess its needed to be backward compatible. We should use new tick format all the way in v3.